var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -105,6 +109,8 @@ class Qt(Package):
     # depends_on("ogg", when='+multimedia')
 
     use_xcode = True
+    # parallel build fails on Sierra
+    parallel = False
```

Comment:
We should use that (and any other system options as well). It will bring compilation time down a bit and let us reuse shared libraries.